### DISCUSSION TITLE
Compile the turbulence model

### DISCUSSION THREAD

**Discussion by cczz8** [mdolab/dafoam#732] [General]
Dear Professor He,

I am currently learning how to compile turbulence models in DAFoam. To understand the process of compiling a turbulence model, I performed the following steps:

I copied and renamed the DAkOmegaSST folder under src/adjoint/DAmodel/DATurbulenceModel to myDAkOmegaSST (only the name was changed).
I added the line DAModel/DATurbulenceModel/myDAkOmegaSST.C to src/adjoint/Make/files_Incompressible.
The compilation was successful.
However, when I tried to use the newly compiled myDAkOmegaSST in the Ramp example, it did not appear in the available DATurbulenceModels types or the RASmodel list.

I would like to understand how to correctly compile and use a custom turbulence model in DAFoam.

**Comment by friedenhe** [comment]
The way you added the new turbo model is not quite correct. You also need to add myDAkOmegaSST to src/adjoints/models. Also, make sure you compile both orig and ADR versions of DAFoam.
